app/views/ansible_variables/index.html.erb

```diff
@@ -1,14 +1,16 @@
 <% title _("Ansible Variables") %>
 
-<% title_actions ansible_proxy_import(hash_for_import_ansible_variables_path),
-  documentation_button('#4.3Variables', :root_url => ansible_doc_url) %>
+<% title_actions display_link_if_authorized(_('New Custom Ansible Variable'), hash_for_new_ansible_variable_path, :class => "btn btn-default"),
+    ansible_proxy_import(hash_for_import_ansible_variables_path),
+    documentation_button('#4.3Variables', :root_url => ansible_doc_url) %>
 
 <table class="<%= table_css_classes 'table-fixed' %>">
   <thead>
     <tr>
       <th class='col-md-6'><%= sort :name, :as => s_('Variable|Name') %></th>
       <th class='col-md-2'><%= sort :ansible_role, :as => s_('Variable|Role') %></th>
-      <th class='col-md-2'><%= _('Type') %></th>
+      <th class='col-md-1'><%= _('Type') %></th>
+      <th class='col-md-1'><%= _('Custom variable?') %></th>
```

Comment:
this is variables table, no need to repeat it here, just say `_('Custom')`

perhaps it would be better to say `Imported`, that is more descriptive, but of course changes the meaning and would need to change on many places, would that be too much to ask? :-)